hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -508,6 +509,21 @@ signWith (TxId h) (PaymentVerificationKey vk, PaymentSigningKey sk) =
     Ledger.Shelley.WitVKey
       (Ledger.asWitness vk)
       (Ledger.signedDSIGN @Ledger.StandardCrypto sk h)
+-- * Extra
+
+-- TODO: Could be offered upstream.
+class ToCtxTx f where
```

Comment:
I understand `CtxTx` and `CtxUTxO` are "blessed" names from the API but those are barely legible: Perhaps we could come up with something else, perhaps`toTxContext`?